tests/unit/mock.py

```diff
@@ -156,10 +159,24 @@ def list_runner_applications_for_org(self, org: str):
         return self._list_runner_applications()
 
     def create_registration_token_for_repo(self, owner: str, repo: str):
-        return RegisterToken({"token": "test registration token"})
+        return RegistrationToken(
+            {"token": "test registration token", "expires_at": "2020-01-22T12:13:35.123-08:00"}
+        )
 
     def create_registration_token_for_org(self, org: str):
-        return RegisterToken({"token": "test registration token"})
+        return RegistrationToken(
+            {"token": "test registration token", "expires_at": "2020-01-22T12:13:35.123-08:00"}
+        )
+
+    def create_remove_token_for_repo(self, owner: str, repo: str):
+        return RemoveToken(
+            {"token": "test registration token", "expires_at": "2020-01-22T12:13:35.123-08:00"}
+        )
+
+    def create_remove_token_for_org(self, org: str):
+        return RemoveToken(
+            {"token": "test registration token", "expires_at": "2020-01-22T12:13:35.123-08:00"}
+        )
```

Comment:
Would it be simpler to have a mock remove token as a constant defined somewhere? :) It seems like it is getting used in a few places.

Reply:
I updated to use a test fixture that generates the token with `secrets.token_hex`.